openrpc.yaml

```diff
@@ -521,6 +521,263 @@ methods:
       schema:
         type: string
         title: QR code string
+  - name: wallet_sendCalls
+    tags:
+      - $ref: '#/components/tags/MetaMask'
+    summary: Sends a batch of calls.
+    description: >-
+      Requests that the wallet submits a batch of calls. Specified by
+      [EIP-5792](https://eips.ethereum.org/EIPS/eip-5792).
+    params:
+      - name: Calls
+        required: true
+        schema:
+          $ref: '#/components/schemas/SendCallsParameter'
+    result:
+      name: Batch result
+      schema:
+        description: An object containing information about the sent batch.
+        type: object
+        properties:
+          id:
+            description: The ID of the batch of calls.
+            type: string
+          capabilities:
+            description: >-
+              Wallets can use this object to attach capability-specific metadata.
+            type: object
+    errors:
+      - code: -32603
+        message: User rejected account upgrade.
+      - $ref: '#/components/errors/InvalidParams'
```

Comment:
We also have the standard rpc InvalidInput error if the version is not set to 2.0.0

Reply:
And we also have the `5720 - There is already a bundle submitted with this id`